<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CWeatherComponent</class>
 <widget class="QWidget" name="CWeatherComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Weather</string>
  </property>
  <property name="currentIndex" stdset="0">
   <number>0</number>
  </property>
  <layout class="QVBoxLayout" name="vl_WeatherComponent">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_weatherControl">
     <property name="title">
      <string>Weather Control</string>
     </property>
     <layout class="QGridLayout" name="gl_WeatherComponent">
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="cb_weatherScenario"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pb_ActivateWeather">
        <property name="text">
         <string>Activate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_weatherDisplay">
     <property name="title">
      <string>Weather Display</string>
     </property>
     <layout class="QVBoxLayout" name="vl_WeatherDisplay">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QFrame" name="fr_Misc">
        <layout class="QHBoxLayout" name="hl_Misc">
         <item>
          <widget class="QCheckBox" name="cb_UseOwnAcftPosition">
           <property name="text">
            <string>Use own aircraft position</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="pb_Coordinate">
           <property name="text">
            <string>coordinate</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbl_Status">
        <property name="text">
         <string>&lt;status will go here&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tw_weatherGrid">
        <widget class="QWidget" name="tb_temperatureLayers">
         <attribute name="title">
          <string>Temperature</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="BlackGui::Views::CTemperatureLayerView" name="tvp_TemperatureLayers">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tb_cloudLayers">
         <attribute name="title">
          <string>Clouds</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="BlackGui::Views::CCloudLayerView" name="tvp_CloudLayers">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tb_windLayers">
         <attribute name="title">
          <string>Winds</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>1</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="BlackGui::Views::CWindLayerView" name="tvp_WindLayers">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Views::CTemperatureLayerView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/temperaturelayerview.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::Views::CCloudLayerView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/cloudlayerview.h</header>
  </customwidget>
  <customwidget>
   <class>BlackGui::Views::CWindLayerView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/windlayerview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
